project/src/main/webapp/events-feed.js

```diff
@@ -3,6 +3,10 @@ $(async function() {
   populateAllEvents(allEvents);
 });
 
+$(document).ready(function(){
```

Comment:
The function above this is already a document.ready one, so the toggle should probably either go straight in there or in a short function called `toggleDropdown` and called from the above function

Reply:
done